wiki/Manual.md

```diff
@@ -1015,43 +1015,29 @@ Examples:
     + good: 0.05, 0.051, 0.0501, 0.0502, ..., 0.0599, 0.06, 0.0601, ...
     - bad: 0.04, 0.049, 0.0499
     ```
-  - *Precision of the amount* should be up to 4 decimal digits:
+  - *Precision of the amount* should be up to 4 digits after dot (0.0001):
     ```diff
     + good: 0.05, 0.051, 0.052, ..., 0.0531, ..., 0.06, ... 0.0719, ...
     - bad: 0.05001, 0.05000, 0.06001
     ```
 
-2. `(market['limits']['price']['min'] == 0.019) && (market['precision']['price'] == 5)`
```

Comment:
the number `(2)` example was a total duplicate of `(1)` example, there is no need to make users read dozens of readmes while 1st example makes it totally clear